e2e_test/bin/async_worker_in_isolate.dart

```diff
@@ -14,7 +14,7 @@ import 'package:e2e_test/forwards_to_isolate_async_worker.dart';
 /// Anyone actually using the facility to wrap a worker in an isolate will want
 /// to use this code to do additional work, for example post processing one of
 /// the output files.
-Future main(List<String> args, SendPort message) async {
+Future main(List<String> args, SendPort? message) async {
```

Comment:
nit: make this optional

```suggestion
Future main(List<String> args, [SendPort? message]) async {
```

Reply:
Done